### DISCUSSION TITLE
Hover mouse to open sidepanel

### DISCUSSION THREAD

**Discussion by DrNopt** [mbnuqw/sidebery#1182] [Ideas]
Would be very nice and easy no navigate if we could hover the mouse on the left/right side to open sidepanel with the bookmarks in it
Same as this Chrome extension: [https://chrome.google.com/webstore/detail/bookmark-sidebar/jdbnofccmhefkmjbkkdkfiicjkgofkdh
](https://chrome.google.com/webstore/detail/bookmark-sidebar/jdbnofccmhefkmjbkkdkfiicjkgofkdh
)

It would do the same function as pressing F1, but just hovering/clicking mouse on the left/right side.
Thank you

**Comment by bedbug1226** [comment]
Totally Agree. Trying to jump off chrome and its one of the extensions that I cant find a matching action for.

Would be nice if sidebery took some inspiration from bookmark sidebar on chrome.

**Comment by furu00** [comment]
https://github.com/lucaspar/firefox-vertical-tabs

👍

**Reply by bedbug1226** [reply]
It would be nice to not have any of the side bar visible though.
